cves/CVE-2015-5964.yml

```diff
@@ -49,7 +49,11 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: Site running on Django could be crashed by a denial of service
+ attack, by accessing a logout page without the proper credentials. This would
+  result in denial of service by allowing a malicious actor to create an
```

Comment:
The way you phrase it here makes it seem as though they gain full access to the system, try being explicit in what an attacker can accomplish using this exploit.

Reply:
Sure, that wasn't my intent, I'll try to rephrase